rank/src/services/test.ts

```diff
@@ -27,73 +25,65 @@ const indexExists = async (client: Client, index: string): Promise<boolean> => {
   }
 }
 
-async function service({
+export async function getClient(queryEnv, cluster, template): Promise<Client> {
+  if (queryEnv === 'candidate' && cluster === 'pipeline') {
```

Comment:
nit: flip the order of conditions, so it's easier to see this is a special instance of the next case

```suggestion
  if (cluster === 'pipeline' && queryEnv === 'candidate') {
```